Search bar relocated

gersongcg opened the pull request:
# Changes

- Theme layout cleanup
- CSS fixes

toddweetman reviewed (commented):
I feel this needs a little more work to be worth changing.

The current means no search bar when scrolling. I don't think that is a bad thing.
The easier solution may be to keep sticky bar to full width and include the searchbox in the sticky component.

Making the header shrink to half page squashes it up a little too much and adding the search bar causes it to stretch (should remain consistent).

I feel we can leave as is for now and when you have more time, work on implementing this better.

gersongcg commented:
@toddweetman I have removed the new structure for the search. However, we need need to update the two files that I have cleaned up. No front-end changes.
